Add beta agreement (#636)
box/box-ios-sdk: docs/usage/getting-started.md

```

__Step 5__: Run the sample app

License
-------

> Any use of this software is governed by the attached [Box SDK Beta Agreement](../../BETA-AGREEMENT.md).  If you do not

Reviewer: ```suggestion
Any use of this software is governed by the attached [Box SDK Beta Agreement](../../BETA-AGREEMENT.md).  By using this software,
```

Reviewer: After reviewing "Proposed Changes to GitHub page of Box iOS SDK by Legal.docx" ( https://cloud.app.box.com/s/0fe26b12k1ekkrzpimrnnnn19d89x320) ... never mind.

> accept the terms of the Box SDK Beta Agreement, you may not use this software.

Reviewer: ```suggestion
you accept the terms of the Box SDK Beta Agreement.
```

Reviewer: After reviewing "Proposed Changes to GitHub page of Box iOS SDK by Legal.docx" ( https://cloud.app.box.com/s/0fe26b12k1ekkrzpimrnnnn19d89x320) ... never mind.
